modules/vector-search/pages/vector-search-index-architecture.adoc

```diff
@@ -0,0 +1,152 @@
+You can choose to prioritize recall or latency with the xref:search:child-field-options-reference.adoc#optimized[Optimized For] setting on your index.
+You can also choose to xref:fine-tune-vector-search.adoc[fine tune your Vector Search queries] to override the default balancing for your index, and change the number of centroids or probes searched in a query. 
+
+The FAISS indexes created for your vector data can be: 
+
+* <<flat,>>
+* <<ivf,>>
+
+The specific type of index used depends on the number of vectors in your dataset: 
+
+|====
+| Vector Count | Index Types | Description
+
+| >=10,000 
+| IVF with scalar quantization
+a| Vectors are indexed with <<ivf,>> indexes and <<scalar-quant,>>.
+
+If xref:search:child-field-options-reference.adoc#optimized[Optimized For] is set to *recall* or *latency*, Vector Search uses 8bit scalar quantization.
+If set to *memory-efficient*, Vector Search uses 4bit scalar quantization.
+
+| >=1000
+| IVF with Flat
+| Vectors are indexed with <<ivf,>> combined with <<flat,>>. 
+Indexes do not use <<scalar-quant,>>. 
+
+| <1000
+| Flat
+| Vectors are indexed with <<flat,>>.
+Indexes do not use <<scalar-quant,>>. 
+|====
+
+[#flat]
+=== FLAT Indexes
+
+The most basic kind of index that Vector Search can use for your vectors is a flat index.
+
+Vector Search uses flat indexes for data that contains less than 1000 vectors.
+
+Flat indexes are a list of vectors. 
+Searches run on a nearest neighbor process, based on examining the query vector against each vector in the index and calculating the distance.
+Results for flat indexes are very accurate, but performance does not scale well as a dataset grows.
+
+If a Vector Search index uses only flat indexes, no training is required - IDs are mapped directly to vectors with exact vector comparisons, with no need for preprocessing or learning on the data.
+
+[#ivf]
+=== Inverted File Index (IVF)
+
+For reduced latency, Vector Search can also use Inverted File Indexes (IVF).
+
+Vector Search uses a combination of IVF and flat indexes for data that contains between 1000 and 9999 vectors.
+For even larger datasets, Vector Search uses IVF indexes with <<scalar-quant,>>.
+
+IVF creates partitions called Voronoi cells in an index. 
+The total number of cells is the *nlist* parameter. 
+
+Every cell has a centroid.
+Every vector in the processed dataset is assigned to a cell that corresponds to its nearest centroid. 
+
+In an IVF index, a Vector Search first tries to find the cell that the query vector belongs to.
+After it knows the cell to search, Vector Search uses another algorithm to find out the exact vector that's closest to the query vector in that cell. 
+
+The result of an IVF index search can be less accurate, as the nearest vector to a query vector can be in a different cell than the chosen cell.
```

Comment:
Let's omit this "can be less accurate" bit because it sounds alarming - especially because the user has no control over this. You can say - `search over centroid indexes is non-exhaustive` .. and the reader can infer the rest.

Reply:
Valid. Changing to "IVF index searches are not exhaustive searches."